packages/grid/x-data-grid-generator/package.json

```
@@ -41,7 +41,7 @@
   },
   "peerDependencies": {
     "@mui/icons-material": "^5.2.5",
-    "@mui/material": "^5.2.7",
+    "@mui/material": "^5.2.8",
```

I would argue that changes here are BCs. I would have expected:
```suggestion
    "@mui/material": "^5.0.0",
```